Makefile

```diff
@@ -152,9 +158,11 @@ local-unit-test: ## Run backend tests in the dev environment
 	@if [ -z "$(path)" ]; then \
         echo "Running all tests"; \
 		docker-compose exec -e DEV_MODE_COOKIES= -T backend bash -c "cd /corpora-data-portal && make container-unittest"; \
+		docker-compose exec -e DEV_MODE_COOKIES= -T processing bash -c "cd /corpora-data-portal && make processing-unittest"; \
```

Comment:
Adding this here, so the processing unit tests run with the rest of the unit tests on push.